packages/ckeditor5-engine/src/model/documentselection.js

```diff
@@ -662,7 +680,9 @@ class LiveSelection extends Selection {
 		} );
 
 		// Update markers data stored by the selection after each marker change.
-		this.listenTo( this._model.markers, 'update', () => this._updateMarkers() );
+		this.listenTo( this._model.markers, 'update', ( evt, marker, oldRange, newRange ) => {
```

Comment:
Could you add a comment here:

```
// This handles only marker changes done through marker operations (not model tree changes).
```

I know this is not connected with your changes but it could be a good tip for someone who is newer to how CKE5 works.

Reply:
done